plugins/outputs/iotdb/iotdb.go

```diff
@@ -0,0 +1,287 @@
+//go:generate ../../../tools/readme_config_includer/generator
+package iotdb
+
+// iotdb.go
+
+import (
+	_ "embed"
+	"errors"
+	"fmt"
+	"math"
+
+	// Register IoTDB go client
```

Comment:
This comment does not provide additional information.

Reply:
All right. Thank you for your careful review.